sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs

```diff
@@ -162,7 +167,7 @@ public virtual Task<Response<BulkRegistryOperationResponse>> CreateIdentitiesWit
                     ImportMode = ExportImportDeviceImportMode.Create
                 });
 
-            return _registryManagerClient.BulkDeviceCrudAsync(registryOperations, cancellationToken);
+            return _devicesRestClient.BulkRegistryOperationsAsync(registryOperations, cancellationToken);
```

Comment:
We can request to change to BulkDeviceCrud. Anyone have any concerns?

Reply:
Can you elaborate?

Reply:
The bulk function does create, update and delete. We have split them up by functionality. Instead of calling them RegistryOperations, I was thinking calling them CrudOperations is better. Autorest automatically generates function names based on the set "operation_id". If that value is updated, it does not affect the service but the name of the function will change. I wanted to know what everyone thinks of the original name vs the updated name.

Reply:
So we call it `RegistryCrudOperations`? I would like to see the name `Registry` in there, just to highlight that there crud operations are on the device identity registry.

Reply:
That would be a good mix.

Reply:
I would never use the word "Crud" in an API name. It is shorthand for us, but I don't think it is appropriate for formal API names.

Reply:
but this is an internal type, is this to maintain good code naming practices everywhere?

Reply:
BulkRegistryRestClient.UpdateRegistry